app/routes/__frontend/events/index.jsx

```diff
@@ -51,6 +51,23 @@ export default function EventsIndex() {
 						</p>
 					</div>
 
+					<h2>Recorded Events</h2>
+					<div className="mb-3 lead">
```

Comment:
- Change to h3
- Remove `lead`
- Let's also add an `<hr />` before and after this entire section

Reply:
I've changed as per request! 😀